Fix EQV bug. "0 EQV 0" will return correct result "1"

Joe7M opened the pull request:
Hi Chris,

EQV works fine but not for 0 EQV 0 which returns 0. Now 0 EQV 0 will return 1.

Best regards, Joerg

chrisws commented:
Hi Joerg,

This looks fine, but could please add some tests for this to samples/distro-examples/tests/eval-test.bas?

I'm just about ready to release 12.26, maybe in a few days.

Joe7M commented:
I created the unit tests. I also included IMP in the unit test and realized that IMP has the same problem: 0 IMP 0 returns 0 instead of 1.  I fixed it the same way as I fixed EQV.

chrisws reviewed (approved):
Looks good
